db/seeds/local_authorities.rb

```diff
@@ -4,7 +4,6 @@
   Rails.root,
   "db",
   "data",
```

Comment:
The other CSV files were moved, but this one was deleted. Was this intentional? (That's what's causing the CI to fail.)